recipes/imgui-sfml/config.yml

```diff
@@ -1,3 +1,7 @@
 versions:
   "2.1":
     folder: all
+  "2.2":
+    folder: all
+  "f35da327e79a00a9b3514fc5ff4de52456f78961":
+    folder: all
```

Comment:
```suggestion
```

Reply:
2.3 came out some time ago, so maybe we should add it too